addons/website_hr_recruitment/models/hr_job.py

```diff
@@ -49,19 +49,19 @@ def _get_default_website_description(self):
             <span class="text-muted small">Days to get an Offer</span>
             <h6>4 Days after Interview</h6>
         """)
-    published_date = fields.Date(compute='_compute_published_date', store=True)
+    published_date = fields.Datetime(compute='_compute_published_date', store=True)
     full_url = fields.Char('job URL', compute='_compute_full_url')
 
+    @api.depends('website_published')
+    def _compute_published_date(self):
+        for job in self:
+            job.published_date = job.website_published and fields.datetime.now()
+
     @api.depends('website_url')
     def _compute_full_url(self):
         for job in self:
             job.full_url = url_join(job.get_base_url(), (job.website_url or '/jobs'))
 
-    @api.depends('website_published')
-    def _compute_published_date(self):
-        for job in self:
-            job.published_date = job.website_published and fields.Date.today()
-
```

Comment:
It could be nice to keep the method in the same place to make the useful diff more readable :)

Reply:
@Bertrand2 Indeed. I'm asking myself how I could have done this ...

Reply:
Thanks for the change! LGTM for the HR part :)